devicon.json

```diff
@@ -1242,6 +1242,23 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "haxe",
+        "tags": [
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#F28820",
```

Comment:
https://haxe.org/foundation/branding.html

![image](https://user-images.githubusercontent.com/8781699/97120148-77682c80-1715-11eb-93f3-e9a4dc936acc.png)

```suggestion
        "color": "#EA8220",
```

Reply:
Ok, let me change it.